src/helpers/countAnswers.ts

```diff
@@ -0,0 +1,17 @@
+import { AnswerEnum } from '../AnswerEnum';
+import { IAnswer } from '../models';
+
+export const countAnswers = (answers: IAnswer[]) => {
```

Comment:
`countAnswers` means that your function returns a number
But it returns something else